sample/AirshipSample/android/gradlew.bat

```diff
@@ -1,3 +1,19 @@
+@rem
+@rem Copyright 2015 the original author or authors.

```

Comment:
do we need the correct year for these apache copyrights headers?

Reply:
this is generated, so no